docs/pages/blog/base-ui-2024-plans.md

```diff
@@ -0,0 +1,104 @@
+This year will see a lot of investment in Base UI as we expand the team ([we're hiring!](/careers/staff-ui-engineer-base-ui/)) and focus hard on a [stable release](https://github.com/mui/material-ui/milestone/46) (tentatively planned for Q3/2024), which will come full of new components, features, and improvements.
+
+Let's walk through some of the things we're cooking up.
+
+## A larger set of components
+
+Base UI today offers a modest set of components and hooks, including some slightly more complex ones such as Autocomplete and Number Input.
+However, we're conscious that many primitive components that would allow a real-world website or application to use Base UI as its foundation are missing.
+Worry no more, as we're focused on shipping the first versions of all of the components listed below for the stable release:
+
+| To-be-added components |                                                                                                |
+| :--------------------- | ---------------------------------------------------------------------------------------------: |
+| Accordion              | [View&nbsp;the&nbsp;GitHub&nbsp;issue&nbsp;→](https://github.com/mui/material-ui/issues/38037) |
+| Alert Dialog           |                [View the GitHub issue&nbsp;→](https://github.com/mui/material-ui/issues/40886) |
+| Checkbox               |                [View the GitHub issue&nbsp;→](https://github.com/mui/material-ui/issues/38036) |
+| Collapsible            |                [View the GitHub issue&nbsp;→](https://github.com/mui/material-ui/issues/40959) |
+| Drawer                 |                [View the GitHub issue&nbsp;→](https://github.com/mui/material-ui/issues/38181) |
+| Radio Group            |                [View the GitHub issue&nbsp;→](https://github.com/mui/material-ui/issues/38038) |
+| Tooltip                |                [View the GitHub issue&nbsp;→](https://github.com/mui/material-ui/issues/38045) |
+
+## Improved customization API
+
+Currently, Base UI components can be customized to your heart's content using the `slots` and `slotProps` props.
+(Read more about them in the "[Overriding component structure](/base-ui/guides/overriding-component-structure/)" guide.)
+
+```tsx
+// Example of the slot prop
+<Select slots={{ listbox: 'ol' }} defaultValue="First option">
+  <Option value="First option">First option</Option>
+  <Option value="Second option">Second option</Option>
+</Select>
+
+// Example of the slotProps prop
+<Badge slotProps={{ badge: { className: 'my-badge' } }} />
+```
+
+This API, while powerful, has proven to be less than ideal in some instances.
+Most notably, it's too lengthy and complicated to write and read when used with libraries such as Tailwind CSS.
+Additionally, since the `slots` and the corresponding `slotProps` are not related to Typescript types, it's possible to introduce bugs or have the compiler complain about valid code.
```

Comment:
Don't trust Grammarly on this one ;)

```suggestion
Additionally, since the `slots` and the corresponding `slotProps` are not related in terms of  Typescript types, it's possible to introduce bugs or have the compiler complain about valid code.
```

Or maybe I phrased it incorrectly? I meant that the types of `slots` and `slotProps` are not related.